src/components/Dialog/docs/DialogWiringExample.tsx

```diff
@@ -50,7 +49,32 @@ export class DialogWiringExample extends React.Component {
           onClose={this._onCloseDialog}
           wiredToSiteColors={isWired}
         >
-          <div className={classes.content}>This is the content!</div>
+          <div className={classes.content} style={{ textAlign: 'center' }}>
```

Comment:
It seems like the style={{ textAlign: 'center' }} is not needed, it's defined in the css file as well :)